openff/bespokefit/executor/services/_settings.py

```diff
@@ -28,7 +28,7 @@ class Settings(BaseSettings):
     BEFLOW_GATEWAY_LOG_LEVEL: str = "error"
 
     BEFLOW_REDIS_ADDRESS: str = "localhost"
-    BEFLOW_REDIS_PORT: int = 6379
+    BEFLOW_REDIS_PORT: int = 6363
```

Comment:
By changing the default port hopefully we're less likely to accidentally connect to someones running redis server unintentionally!

Reply:
Great idea, this should be much safer.